ci-operator/step-registry/openshift/e2e/aws/csi/efs/extended/openshift-e2e-aws-csi-efs-extended-workflow.yaml

```diff
@@ -14,4 +14,4 @@ workflow:
       TEST_FILTERS: "~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;~StagerunOnly;~LSO&;STORAGE&"
       FORCE_SUCCESS_EXIT: "no"
   documentation: |-
-    The Openshift E2E AWS EFS CSI extended workflow executes the `openshift csi extended end-to-end test suite` on AWS EFS CSI driver and operator.
```

Comment:
If the workflow name changed to `openshift-e2e-aws-csi-efs-sts-extended` shouldn't this file name change as well?

Reply:
Yeah, I added new workflow `openshift-e2e-aws-csi-efs-sts-extended`, sorry for that the documentation of `openshift-e2e-aws-csi-efs-extended-workflow` should not updated.